keyless/circuit-tests/src/lib.rs

```diff
@@ -24,9 +24,10 @@ impl TestCircuitHandle {
         let include_root_dir = cargo_manifest_dir.join("../circuit-data/templates");
         let src_circuit_path = include_root_dir.join("tests").join(file_name);
         let tmp_circuit_path = dir.path().to_owned().join("circuit.circom");
-        let tmp_circuit_file = File::create(&tmp_circuit_path)?;
-        fs::copy(&src_circuit_path, &tmp_circuit_path)?;
-        let output = Command::new("circom").args(&[
+        // Rex: why is this variable never used?
+        let _tmp_circuit_file = File::create(&tmp_circuit_path)?;
```

Comment:
@zjma Why can't we just do `File::create(&tmp_circuit_path)?;` without assigning to a variable here? Does something happen when the variable goes out of scope?

Reply:
i think it doesn't have to stay in the scope and i forgot to lint my migration PR 🤦

Reply:
let's see what the CI check say